GH Action – Comment built image names and tags to a PR workflow failing

### Context

I have updated the `requirements.txt` file in the Handbook Authoring image in PR https://github.com/2i2c-org/community-showcase/pull/49.

The workflows for generating the build matrix, as well as building and pushing image to Quay.io, appear to be [working fine](https://github.com/2i2c-org/community-showcase/actions/runs/7486100559). However, the subsequent workflow appears to be [failing](https://github.com/2i2c-org/community-showcase/actions/runs/7486169087) with the error message:

```
The template is not valid. .github/workflows/comment-built-images-to-pr.yaml (Line: 25, Col: 25): A mapping was not expected
```

### Proposal

_No response_

### Updates and actions

_No response_

I have made progress. I have solved the issue in the GitHub workflow file, and now have a new error of:

```
Traceback (most recent call last):
  File "/home/runner/work/community-showcase/community-showcase/scripts/comment-image-name-to-pr.py", line 92, in <module>
    with open(f"name.txt") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'name.txt'
Error: Process completed with exit code 1.
```

--- a/images/test-image/requirements.txt
+++ b/images/test-image/requirements.txt
@@ -1,2 +1,3 @@
 numpy
 pandas
+matplotlib

